examples/test/test_resnet_ptq_cpu.py

```diff
@@ -24,7 +24,7 @@ def setup():
     # retry since it fails randomly
     retry_func(run_subprocess, kwargs={"cmd": "python prepare_model_data.py", "check": True})
 
-    set_azure_identity_logging()
+    # set_azure_identity_logging()
```

Comment:
Any reason to disable this logging?

Reply:
the azure identity logs are too verbose and makes it hard to see the relevant information. This was added by mike when we had azure credential issues due to permissions but it's not needed anymore.

Reply:
Ic, maybe we can try to remove it later. Or you want to keep it to remind the follwoing developer to use it for debugging purpose?

Reply:
just kept it for now if we need to debug similar issues but I can remove it in the follow up.